### PR TITLE
Fix #278: clear syntax error messages when code is reset.

### LINKED ISSUE #278: Resetting feedback window does not remove syntax error
Expected behavior: When resetting the feedback window, displayed syntax errors are also cleared.

Observed behavior: Resetting the feedback window does not clear displayed syntax errors.

Steps to reproduce: Generate a syntax error and display details. Reset feedback window and note that syntax error is not cleared.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: client/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -52,17 +52,17 @@ tie.directive('learnerView', [function() {
...
                   </p>
                 </div>
-                <div class="tie-feedback-syntax-error">
-                  <a href class="tie-feedback-syntax-error-link",
-                      ng-click="toggleSyntaxErrorHint()",
-                      ng-show="syntaxErrorFound">
-                    {{isSyntaxErrorShown ? 'Hide error details' : 'Display error details'}}
-                  </a>
-                </div>
+
```
Comment (rabidbit) [inline client/components/LearnerViewDirective.js]: Remove empty line.

File: client/components/LearnerViewDirective.js [hunk 2]
```diff
@@ -52,17 +52,17 @@ tie.directive('learnerView', [function() {
...
+                   ng-click="toggleSyntaxErrorHint()">
+                  {{isSyntaxErrorShown ? 'Hide error details' : 'Display error details'}}
+                </a>
                 <br>
-                <span class = "tie-feedback-error-string", ng-show="isSyntaxErrorShown">
+                <span class="tie-feedback-syntax-error-string" ng-if="syntaxErrorString"
+                      ng-show="isSyntaxErrorShown">
                   {{syntaxErrorString}}
                 </span>
+
```
Comment (rabidbit) [inline client/components/LearnerViewDirective.js]: Remove empty line.

**Comment by rabidbit** [comment]
When I try to access the code for the commit, I get a 404. Not sure if it's just me.

File: client/components/LearnerViewDirective.js (hunk 2 above)
Comment (seanlip) [inline client/components/LearnerViewDirective.js]: Done.

File: client/components/LearnerViewDirective.js (hunk 1 above)
Comment (seanlip) [inline client/components/LearnerViewDirective.js]: Done.

**Comment by seanlip** [comment]
Thanks! Comments addressed, PTAL.

**Comment by seanlip** [comment]
Travis isn't triggering. Closing and reopening.

**Pull request merged**